ios-plugin/commons-ios/src/main/java/io/ecocode/ios/rules/JSONRulesDefinition.java

```diff
@@ -1,6 +1,6 @@
 /*
- * SonarQube Apple Plugin - Enables analysis of Swift and Objective-C projects into SonarQube.
- * Copyright © 2022 inside|app (contact@insideapp.fr)
+ * ecoCode iOS plugin - Help the earth, adopt this green plugin for your applications
+ * Copyright © 2022 CNumR (https://www.ecocode.io/)
```

Comment:
not CNumR but Green Code Initiative (same review note in all files)

Reply:
Hi,
Actually the header is generated automatically from pom data.
In parent pom.xml organization is still CNumR.
I will update it.
Also url is still https://collectif.greenit.fr, should I change it as well ?

Reply:
We have a dedicated PR for this. We will change it later.